docs/reference/commandline/service_create.md

```diff
@@ -112,6 +112,26 @@ dmu1ept4cxcf  redis   replicated  1/1       redis:3.0.6
 a8q9dasaafud  redis2  global      1/1       redis:3.0.6
 ```
 
+#### Create a service using an image on a private registry
+
+If your image is available on a private registry which requires login, use the
+`--with-registry-auth` flag with `docker service create`, after logging in. If
+your image is stored on `registry.example.com`, which is a private registry, use
+a command like the following:
+
+```bash
+$ docker login registry.example.com
+
+$ docker service  create \
+  --with-registry-auth \
+  --name my_service \
+  registry.example.com/acme/my_image:latest
+```
+
+This passes the login token from your local client to the swarm nodes where the
+service is deployed, using the encrypted WAL logs. With this information, the
+nodes are able to log into the registry and pull the image.
+
```

Comment:
As a follow up, perhaps we should explain that these credentials are _stored_ in the swarm (and are also used when scaling a service or if tasks are rescheduled). One thing people trip over is that some registries use a short-lived token for authentication, which means that if a task is rescheduled, the token may be expired, so pulling the image fails.